docs/sources/setup-grafana/configure-security/configure-authentication/saml/index.md

```diff
@@ -373,6 +373,20 @@ SAML's single logout feature allows users to log out from all applications assoc
 `HTTP-Redirect` and `HTTP-POST` bindings are supported for single logout.
 When using `HTTP-Redirect` bindings the query should include a request signature.
 
+#### Configure single logout
+
+{{% admonition type="note" %}}
```

Comment:
I always find those sections starting with "disclaimers/notes" less readable to be honest, as the first I want to see when I read "Configure single logout" is to learn how to configure single logout. In other words, I'd suggest to move the note down the next paragraph.